src/main/kotlin/lotto/model/LottoResult.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.model
+
+const val MINIMUM_MATCHES = 3
+
+class LottoResult(private val winningNumbers: List<Int>, private val tickets: List<TicketModel>) {
+    fun calculateResult(): Map<Int, Int> {
+        return tickets.groupingBy { ticket ->
+            ticket.numbers.count { it in winningNumbers }
+        }
+            .eachCount()
+            .filterKeys { it >= MINIMUM_MATCHES }
+    }
+
+    fun totalPrize(): Int{
+        val prizeMap = mapOf(
+            3 to 5_000,
+            4 to 50_000,
+            5 to 1_500_000,
+            6 to 2_000_000_000
+        )
```

Comment:
It's not necessary to create `Map` instance every time the `totalPrize()` is called!

How about refactoring them using enum classes? ☺️ 
https://kotlinlang.org/docs/enum-classes.html